monitoring/monitorlib/geoawareness_automated_testing/api.py

```diff
@@ -1,12 +1,25 @@
 from enum import Enum
 from typing import List, Optional
 from implicitdict import ImplicitDict, StringBasedDateTime
+from uas_standards.eurocae_ed269 import (
+    UomDimensions,
+    VerticalReferenceType,
+    Restriction,
+)
 
 SCOPE_GEOAWARENESS_TEST = "geo-awareness.test"
 
 # Mirrors of types defined in geo-awareness automated testing API
 
 
+class Position(ImplicitDict):
```

Comment:
We don't have to migrate all the existing data types right away, but could we import any of the new ones from `uas_standards.interuss.automated_testing.geo_awareness.v1.api`?

Reply:
All data types should have been migrated by now.